doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-vscode-extension.adoc

```diff
@@ -3,6 +3,8 @@
 
 {PRODUCT} provides a *Red Hat Business Automation Bundle* VSCode extension that enables you to design Decision Model and Notation (DMN) decision models, Business Process Model and Notation (BPMN) 2.0 business processes, and test scenarios directly in VSCode. {PRODUCT} also provides individual *DMN Editor* and *BPMN Editor* VSCode extensions for DMN or BPMN support only, if needed.
 
+IMPORTANT: The VSCode extension is a client-side editor that is partially compatible with the {CENTRAL} features. The features such as importing projects from {CENTRAL} are not supported in the VSCode. If you want to use such features in the VSCode, you can raise a request for a required feature. 
```

Comment:
We should imply editors not the extension, so something like `The editors provided by VSCode extension are only partially compatible with the editors in the business central and several features are not supported.` 
Also, could you please work in something like:
`We recommend using VSCode extension for building new projects.`

Reply:
+1 for such adjustment

Reply:
Done, thanks

Reply:
@domhanak, @porcelli, I have incorporated the suggestion and also added information related to creating maven project. Please take a look one more time and share your feedback